deepchecks/recommender/checks/sample_performance.py

```diff
@@ -59,24 +54,29 @@ def __init__(self,
 
     def run_logic(self, context: Context, dataset_kind) -> CheckResult:
         """Run check."""
+        if self.n_samples ==None:
```

Comment:
Why is this special treatment needed?

Reply:
Fixed.
https://github.com/deepchecks/deepchecks/pull/2610/commits/75f3d5b2e8ba59a47348f57e70d2124f2815da5e